internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: t-add-fbc-fragment-to-index-image
   labels:
-    app.kubernetes.io/version: "0.1"
+    app.kubernetes.io/version: "0.1.1"
```

Comment:
It's a new functionality, so probably 0.2.0?

Reply:
fixed